bin/update_minio_image_tag.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+SOURCE_REPO="cwm-keda-external-scaler"
+TARGET_REPO="cwm-worker-deployment-minio"
+TARGET_FILE="values.yaml"
+
+echo "Updating $SOURCE_IMAGE's image tag in $TARGET_REPO's Helm Chart (values.yaml)"
+
+echo "Setting up GitHub credentials"
+DEPLOY_KEY_FILE="cwm_worker_deploy_key_file"
+echo "$DEPLOY_KEY" > $DEPLOY_KEY_FILE
+chmod 400 $DEPLOY_KEY_FILE
+export GIT_SSH_COMMAND="ssh -i $(pwd)/$DEPLOY_KEY_FILE -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
+git config --global user.name "$SOURCE_REPO CI"
+git config --global user.email "$SOURCE_REPO-ci@localhost"
+
+echo "Cloning $TARGET_REPO to update image tag"
+git clone git@github.com:CloudWebManage/cwm-worker-deployment-minio.git
+
+cd $TARGET_REPO/helm
+IMAGE="docker.pkg.github.com/cwm-keda-external-scaler/iamazeem/cwm-keda-external-scaler"
+IMAGE_WITH_SHA=$IMAGE:$GITHUB_SHA
+sed -i "s#$IMAGE#$IMAGE_WITH_SHA#" ./$TARGET_FILE
```

Comment:
I don't think it will work this way, it will only work for the first update, on next update it will break

it should work the same as it does for the logger image, with an image file: https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/main/helm/cwm-worker-logger.image
and code in the minio repo which updates the helm chart before publishing: https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/main/.github/workflows/ci.yml#L51

Reply:
Tested multiple times with the updated regex and it is working:

```shell
sed -i "s#$IMAGE.*#$IMAGE_WITH_SHA#" ./$TARGET_FILE
```

Is this fine?
Or, is there a specific use-case for a separate file?

Reply:
It makes it harder to develop and test, because you need to check what is the SHA1 of the image to build it for testing / development / in the minio CI

Reply:
Currently for the logger, the values.yaml has the image name without SHA, so it is always latest and that is what we build and test with. We add the SHA only in the published helm chart